app/main/forms.py

```diff
@@ -28,7 +28,7 @@ class CookiesForm(FlaskForm):
 
 class DownloadForm(FlaskForm):
     file_format = SelectField(
-        "File Format",
+        "File type",
```

Comment:
It seems to be referred to as File Format everywhere else, is it best to keep it consistent?

Reply:
This is how it is written in the current design, and as the old download page will be deprecated once this piece of work is done I've opted for "File type".